File: C/Lista_01(Estrutura_Sequencial)/Consumo.c
#include <stdio.h>

int main () {

    int distanciaPercorrida;

    printf("Distancia percorrida: ");
    scanf("%d", &distanciaPercorrida);

    double combustivelGasto;

    printf("Combustivel gasto: ");
    scanf("%lf", &combustivelGasto);

    double consumoMedio = distanciaPercorrida / combustivelGasto;

    printf("Consumo medio = %.3lf", consumoMedio);

}

File: Java/Lista_01(Estrutura_Sequencial)/Consumo.java
import java.util.Locale;
import java.util.Scanner;

public class Consumo {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner s = new Scanner(System.in);
		
		System.out.print("Distancia percorrida: ");
		double distancia = s.nextDouble();
		
		System.out.print("Combustivel gasto: ");
		double combustivel = s.nextDouble();
		
		double consumoMedio = distancia / combustivel;
		
		System.out.printf("Consumo medio: %.3f", consumoMedio);
		
		s.close();
		
	}

}
